### PR TITLE
Refactor contributors and sponsors

### PR DESCRIPTION
**Pull request by saltcod**
Port, tailwind-ify and typescriptify the /docs/oss page

### CODE REVIEW

File: apps/docs/components/GithubCard.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
```
Comment (pucho) [inline apps/docs/components/GithubCard.tsx:2]: Do this cards need a different background? Or are they supposed to blend with the site color?
Screen Shot 2022-11-07 at 11 49 56

**Review by pucho — CHANGES_REQUESTED** [review]
Does the link on the footer have to point to {url}/oss or should it point to {url}/docs/oss?

File: apps/docs/components/GithubCard.tsx (hunk 1 above)
Comment (saltcod) [reply apps/docs/components/GithubCard.tsx:2]: aah good catches. Updated both.

**Review by pucho — APPROVED** [review]
Looks good

**Pull request merged**